.circleci/config.yml

```diff
@@ -51,14 +51,17 @@ jobs:
       - run:
           name: check go fmt
           command: |
-            files=$(go fmt ./...)
+            files="$(go fmt ./... ; (cd api && go fmt ./... | sed 's@^@api/@') ; (cd sdk && go fmt ./... | sed 's@^@sdk/@'))"
```

Comment:
I assume this is for forward compatibility with Go 1.13 where ./api/... and ./sdk/... from the root directory don't work as expected?

Reply:
Apparently not:
```
$ go version
go version go1.12.13 linux/amd64
$ go fmt ./api/... ./...
go: not formatting packages in dependency modules
```

Reply:
You have to cd into the module for the fmt command to work:
```
$ go fmt ./api/... 
go: not formatting packages in dependency modules
$ (cd api && go fmt ./... )
```